rules/php74/tests/Rector/Property/TypedPropertyFromStrictConstructorRector/Fixture/iterable_interface_var_tag.php.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Rector\Php74\Tests\Rector\Property\TypedPropertyFromStrictConstructorRector\Fixture;
+
+use DateTimeInterface;
+
+final class IterableInterfaceVarTag
+{
+    /**
+     * @var iterable|DateTimeInterface[]
+     */
+    private $dates;
+
+    /**
+     * @param DateTimeInterface[] $dates
+     */
+    public function __construct(iterable $dates)
+    {
+        $this->dates = $dates;
+    }
+}
+?>
+-----
+<?php
+
+namespace Rector\Php74\Tests\Rector\Property\TypedPropertyFromStrictConstructorRector\Fixture;
+
+use DateTimeInterface;
+
+final class IterableInterfaceVarTag
+{
+    /**
+     * @var iterable|DateTimeInterface[]
```

Comment:
@TomasVotruba Question: Is there also a Rector that can transform `iterable|DateTimeInterface[]` to `iterable<DateTimeInterface>`? (same for `array`).

Reply:
Hi, it's possible, but that woud require a new rule.